cmd/nfd-master/main.go

```diff
@@ -149,6 +147,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 			" DEPRECATED: will be removed in a future release along with the deprecated gRPC API.")
 	flagset.IntVar(&args.MetricsPort, "metrics", 8081,
 		"Port on which to expose metrics.")
+	flagset.IntVar(&args.GrpcHealthPort, "health", 8082,
```

Comment:
nit: grpc-health ?

nfd is moving away from grpc for master <-> worker comm. so we might want to save the "health" flag for something else :)

Reply:
This is just the health endpoint, I don't think that this has something todo with the grpc master<->worker communication, but I am not that deep into the code base

Reply:
Agree with @adrianchiris , let's clearly indicate that is grpc-health